File: Assignment Programs/Assignment23/program2.c
#include<stdio.h>

void struprX(char *str)
{
	if(str == NULL)
	{
		return;
	}
	while(*str != '\0')
	{
		if((*str>='a') && (*str<='z'))
		{
			*str = *str-32;;
		}
		str++;
	}
} 
int main()
{
	char arr[20];
	printf("Enter string\n");
	scanf("%[^'\n']s",arr);
	struprX(arr);
	printf("Modified string is %s\n",arr);
	return 0;
}

File: Assignment Programs/Assignment54/program2.java
import java.util.Scanner;

class MyArray
{
	public void ArrayReplace(char arr[])
	{
		for(int i=0;i<arr.length;i++)
		{
			if((arr[i] >= 'a') && (arr[i]<='z'))
			{
				arr[i] = (char)(arr[i]-32);
			}
		}
		System.out.println("Modified array is :");
		for(int i = 0;i<arr.length;i++)
		{
			System.out.println(arr[i]);
		}
	}
}

class program2
{
	public static void main(String args[])
	{
		int size = 0;
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter the size of array");
		size = sobj.nextInt();
		
		char Arr[] = new char[size];
		System.out.println("Enter the elements of array");
		for(int i=0;i<size;i++)
		{
			Arr[i] = sobj.next().charAt(0);
		}
		MyArray mobj = new MyArray();
		mobj.ArrayReplace(Arr);
	}
}